cmd/nerdctl/image_prune_test.go

```diff
@@ -40,6 +40,30 @@ func TestImagePrune(t *testing.T) {
 	assert.NilError(t, err)
 	defer os.RemoveAll(buildCtx)
 
+	base.Cmd("build", buildCtx).AssertOK()
+	base.Cmd("build", "-t", imageName, buildCtx).AssertOK()
+	base.Cmd("images").AssertOutContainsAll(imageName, "<none>")
+
+	base.Cmd("image", "prune", "--force").AssertNoOut(imageName)
+	base.Cmd("images").AssertNoOut("<none>")
+	base.Cmd("images").AssertOutContains(imageName)
+}
+
+func TestImagePruneAll(t *testing.T) {
+	testutil.RequiresBuild(t)
+
+	base := testutil.NewBase(t)
+	defer base.Cmd("builder", "prune").Run()
```

Comment:
Can you replace Run with AssertOk. If possible in the previous test as well.

Reply:
I see that the vast majority of commands executed with defer are using `Run`, and that is also what I have been doing. For other commands, I have been using `AssertOK`. Perhaps we could uniformize this approach throughout the entire project later.

Updated.

Reply:
It should be updated; we just change as we go; sometimes Run fails; and our  tests get flaky